matrix/stdlib_imports.hpp

```diff
@@ -15,6 +15,10 @@
 
 namespace matrix {
 
+#if !defined(FLT_EPSILON)
+#define FLT_EPSILON     __FLT_EPSILON__
```

Comment:
@TSC21 this is in `float.h`.

Reply:
Including `float.h` resulted on definition conflicts with the other platforms. Since we just need `FLT_EPSILON` definition, it made more sense to me just include this definiotn.

Reply:
If including a standard c header caused conflicts there are other problems.